ViteMaDose/ViewModels/CentresList/CentresListViewModel.swift

```diff
@@ -70,6 +77,8 @@ class CentresListViewModel {
     var county: County
     var vaccinationCentres: VaccinationCentres?
 
+    var sort: CentresSortOrder = .auPlusVite
```

Comment:
**Which one is the default?** Cf. the other comment.

Reply:
`.auPlusProche` is the default one, I need to correct that.

Reply:
```suggestion
    var sort: CentresSortOrder = .auPlusProche
```